src/lib/sharing.ts

```diff
@@ -21,7 +21,11 @@ export async function shareUrl(url: string) {
   } else {
     // React Native Share is not supported by web. Web Share API
     // has increasing but not full support, so default to clipboard
-    setStringAsync(url)
-    Toast.show(t`Copied to clipboard`, 'clipboard-check')
+    copyUrl(url)
   }
 }
+
+export function copyUrl(url: string) {
+  setStringAsync(url)
```

Comment:
question: `setStringAsync` could fail. Should we make this function async and await the `setStringAsync(url)` call to ensure we actually copy the url before showing the toast?